snmp/datadog_checks/snmp/config.py

```diff
@@ -2,6 +2,7 @@
 # All rights reserved
```

Comment:
It seems that the builder might changed during oid resolution and elsewhere.

https://github.com/etingof/pysnmp/blob/v4.4.12/pysnmp/smi/rfc1902.py#L380-L381

A bit concerned about the possible concurrency issues.